esrally/driver/runner.py

```diff
@@ -1103,6 +1107,101 @@ def __repr__(self, *args, **kwargs):
         return "delete-data-stream"
 
 
+class CreateComponentTemplate(Runner):
+    """
+    Execute the `PUT index template API <https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-component-template.html>`_.
```

Comment:
Nit: In the docs we call it the "Put component template API", not the "PUT index template API".

Reply:
Agreed lets be consistent.